lib/puppet/provider/ec2_vpc/v2.rb

```diff
@@ -0,0 +1,94 @@
+require_relative '../../../puppet_x/puppetlabs/aws.rb'
+
+Puppet::Type.type(:ec2_vpc).provide(:v2, :parent => PuppetX::Puppetlabs::Aws) do
+  confine feature: :aws
+
+  mk_resource_methods
+  remove_method :tags=
```

Comment:
Why is this here? Shouldn't users be able to set tags? If not, should these be a read-only property, or not a property?